### ISSUE TITLE
OMDB API key

### ISSUE THREAD

**Issue by togatown** [michaldrabik/showly-2.0#821]
I have a recently issued free OMDB API key and ratings weren't loading.

The error in the log shows an invalid API key. 

https://private.omdbapi.com/?i=tt0112095&apikey=APIKEY&tomatoes=true 

I requested several more free keys using different email addresses and all of them say they are invalid.

Changing the subdomain to www fixes the issue.

https://www.omdbapi.com/?i=tt0112095&apikey=APIKEY&tomatoes=true

**Comment by michaldrabik** [comment]
Yeah I honestly don't remember why is the private used. For free keys it might be invalid. Sent a message to api owner to clarify.
